File: python/ex2.py

##
##Exercise 2
##
##Write a program that asks the user to provide
##different numbers of banknotes and coins,
##from €50 to €1.
##Then output the total sum in euros. For example:
##
##Enter number of 50 euro banknotes: 10
##Enter number of 20 euro banknotes: 20
##Enter number of 10 euro banknotes: 30
##Enter number of 5 euro banknotes: 40
##Enter number of 2 euro coins: 50
##Enter number 1 euro coints: 60
##You have 1560 euros.
#################################


sum1=0
sum_total=0
list1=[50,20,10,5,2,1]

for i in list1:
        
    sum1=int(input("Enter number of "+str(i)+" euro banknotes: "))
    b=sum1*i
    
    sum_total=b+sum_total

print("You have ",sum_total," euros.")

File: java/Ex2.java
import java.util.Scanner;
/*
 * Ex2.java
 * 
 * Copyright 2016 bill kabb <billkabb@manjaro_pc>
 * 
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.
 * 
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 * 
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 * MA 02110-1301, USA.
 * 
 * 
 */


public class Ex2 {
	
	public static void main (String[] args) {
		int count_temp = 0;
		int count = 0;
		int [] banknotes = {50,20,10,5,2,1};
		for(int x : banknotes)
		{
			System.out.print("enter num of "+x+" banknotes: ");
			Scanner in = new Scanner(System.in);
			int i = in.nextInt();
			count_temp=x*i;
			count+=count_temp;
		}
		System.out.println("You have "+count+" euros.");
	}
}

